lib/bitcoin/fee/SlidingWindowQuantileStoreInitializer.ts

```diff
@@ -0,0 +1,109 @@
+import ISlidingWindowQuantileStore from '../interfaces/ISlidingWindowQuantileStore';
+import ProtocolParameters from '../ProtocolParameters';
+import RunLengthTransformer from './RunLengthTransformer';
+import QuantileInfo from '../models/QuantileInfo';
+import ValueApproximator from './ValueApproximator';
+
+/**
+ * We want to insert some static data in the quantile db to bootstrap the calculation. This class
+ * encapsulates the functionality to do that.
+ */
+export default class SlidingWindowQuantileStoreInitializer {
+
+  private constructor (
+    private genesisBlockNumber: number,
+    private groupSizeInBlocks: number,
+    private windowSizeInGroup: number,
+    private sampleSizePerGroup: number,
+    private initialQuantileValue: number,
+    private mongoDbStore: ISlidingWindowQuantileStore) {
+  }
+
+  /**
+   * Initializes the database if it is empty.
+   *
+   * @param genesisBlockNumber The genesis block.
+   * @param mongoDbStore The data store.
+   */
+  public static async initializeDatabaseIfEmpty (
+    genesisBlockNumber: number,
+    valueApproximator: ValueApproximator,
+    mongoDbStore: ISlidingWindowQuantileStore) {
+
+    // The quantile value below is what we have decided to be the value for the initial
+    // static value.
+    const quantile = 25000;
+    const normalizedQuantile = valueApproximator.getNormalizedValue(quantile);
+
+    const dataInitializer = SlidingWindowQuantileStoreInitializer.createInstance(
+      genesisBlockNumber,
+      normalizedQuantile,
+      mongoDbStore);
+
+    await dataInitializer.addDataIfNecessary();
+  }
+
+  private static createInstance (

```

Comment:
Nit: this method seems to be unnecessary since it is private.

Reply:
This is really only to help with testing. The `initializeDatabaseIfEmpty` calls this function and I can mock the output of this function when testing the `initializeDatabaseIfEmpty` function.

Reply:
I see! Subtle, is it possible to add a comment to say this is for testing only? In case another smart ass like me comes along and thinks he knows better.

Reply:
Fair. Will add a comment for calrity.